Sources/Transmute/transmute().swift

```diff
@@ -70,6 +70,15 @@ public func transmute(rootPackage: Package, externalPackages: [Package]) throws
 
     let modules = recursiveDependencies(packages.flatMap{ map[$0] ?? [] })
     let externalModules = recursiveDependencies(externalPackages.flatMap{ map[$0] ?? [] })
+    
+    let (testProducts, nonTestProducts) = products.partition{
```

Comment:
Tried, not working. I guess equality operator doesn't work if a enum has associated values